rmw_implementation/CMakeLists.txt

```diff
@@ -15,67 +15,55 @@ find_package(ament_cmake REQUIRED)
 
 find_package(rcutils REQUIRED)
 
-# provides FindPoco.cmake and Poco on platforms without it
-find_package(poco_vendor)
-find_package(Poco COMPONENTS Foundation)
 find_package(rmw_implementation_cmake REQUIRED)
 
 if(BUILD_TESTING)
   find_package(ament_lint_auto REQUIRED)
   ament_lint_auto_find_test_dependencies()
 endif()
 
+find_package(rmw REQUIRED)
```

Comment:
Why is this added here? The following function `get_default_rmw_implementation` is provided by `rmw_implementation_cmake` which was found earlier. If it is needed further down maybe moving it closer to where it is used?

Reply:
Ah you're right.

Reply:
Wait... where does this even need the dependency on rmw? It seems like we don't even use it

Reply:
nevermind. I see